Examples/Scripts/FreeSurferPipelineBatchNHP.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash 
+set -e
+# Requirements for this script
+#  installed versions of: FSL6.0.4 or higher , FreeSurfer (version 6.0 or higher) , gradunwarp (python code from MGH)
+#  environment: FSLDIR , FREESURFER_HOME , HCPPIPEDIR , CARET7DIR , PATH (for gradient_unwarp.py)
+
+usage () {
+echo "Usage: $0 <StudyFolder> <SubjectID> <T2w type> <SPECIES> <RunMode>"
+echo "    Runmode: 1 - 3"
+exit 1
+}
+[ "$5" = "" ] && usage
+
+StudyFolder=$1
+Subjlist=$2
+T2wType=$3
+SPECIES=$4
+RunMode=$5
+
+#put the full path to your edited version of SetUpHCPPipeline.sh here
+EnvironmentScript="${HOME}/projects/Pipelines/Examples/Scripts/SetUpHCPPipeline.sh"
+source $EnvironmentScript
+
+# Log the originating call
+echo "$@"
+
+#if [ X$SGE_ROOT != X ] ; then
+#    QUEUE="-q long.q"
+#fi
+
+PRINTCOM=""
+#PRINTCOM="echo"
+
+
+echo "#### Environments ####################################################################"
+echo Host: `hostname`
+echo PATH: $PATH
+echo LD_LIBRARY_PATH: $LD_LIBRARY_PATH
+echo CUDA_VISIBLE_DEVICES: $CUDA_VISIBLE_DEVICES
+echo FREESURFER_HOME=$FREESURFER_HOME
+echo PERL
+which perl
+perl -V | tail -11
```

Comment:
It doesn't look like any of these scripts use perl.  Why is this here?

Reply:
In fact, I don't think the Batch script really needs to report anything about its environment.  What matters is the environment the pipeline sees when it is launched.